js/spbc-common.min.js

```diff
@@ -1,2 +1,2 @@
-function spbc_timestamp(){return Math.floor(+new Date/1e3)}function spbcHighlightElement(e,t){t=t-1||0,jQuery("#"+e).addClass("spbc_highlighted"),jQuery("#"+e).animate({outlineColor:"rgba(255, 0, 0, 1.0)"},400,"linear",function(){jQuery("#"+e).animate({outlineColor:"rgba(255, 0, 0, 0.0)"},400,"linear",function(){0<t?spbcHighlightElement(e,t):jQuery("#"+e).removeClass("spbc_highlighted")})})}function spbc_sendAJAXRequest(t,o,n){var e=o.notJson?"text":"json",r=o.button||null,s=o.spinner||null,l=o.progressbar||null,i=o.callback||null,u=o.notJson||null,a=o.timeout||15e3,n=n||null,c=o.context||this;r&&(r.setAttribute("disabled","disabled"),r.style.cursor="not-allowed"),s&&jQuery(s).css("display","inline"),t.security=spbcSettings.ajax_nonce,t.no_cache=Math.random();var d=o.success||function(e){r&&(r.removeAttribute("disabled"),r.style.cursor="pointer"),s&&jQuery(s).css("display","none"),(e=!u&&"object"!=typeof e?JSON.parse(e):e).error?(setTimeout(function(){l&&l.fadeOut("slow")},1e3),p(e.error)):i&&i(e,t,o,n)},b=o.error||function(e,t,o){r&&(r.removeAttribute("disabled"),r.style.cursor="pointer"),s&&jQuery(s).css("display","none"),console.log("%c APBCT_AJAX_ERROR","color: red;"),console.log(t),console.log(o),console.log(e),p(t)},p=o.errorOutput||function(e){};jQuery.ajax({context:c,type:"POST",url:spbcSettings.ajaxurl,data:t,dataType:e,success:d,error:b,timeout:a})}
```

Comment:
Why the "common" was minimized? I haven't seen any changes.

Reply:
it was fixed earlier